src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        int index = random.nextInt(RandomFigure.values().length);
+        RandomFigure randomFigure = RandomFigure.values()[index];
+        Figure figure;
+        String randomColor = new ColorSupplier().getRandomColor();
+        switch (randomFigure) {
+            case CIRCLE:
+                double randomRadius = random.nextDouble();
+                figure = new Circle(randomRadius, randomColor);
+                break;
```

Comment:
create private method for preparing Figures
```suggestion
                return prepareCircle();
```